### ISSUE TITLE
68. torch.max()

### ISSUE THREAD

**Issue by CHH3213** [CHH3213/Note-Ubuntu_CHH3213#68]
## 形式：`torch.max(input) → Tensor`

返回输入tensor中所有元素的最大值：
```
a = torch.randn(1, 3)
>>0.4729 -0.2266 -0.2085
torch.max(a) #也可以写成a.max()
>>0.4729
```
## 形式： `torch.max(input, dim, keepdim=False, out=None) -> (Tensor, LongTensor)`
按维度dim 返回最大值，并且返回索引。
- `torch.max(a,0)`返回每一列中最大值的那个元素，且返回索引（返回最大元素在这一列的行索引）。返回的最大值和索引各是一个tensor，一起构成元组(Tensor, LongTensor)
- `torch.max(a,1)`返回每一行中最大值的那个元素，且返回其索引（返回最大元素在这一行的列索引）

示例
```
a = torch.tensor([[3, 5], [2, 3], [6, 7]])
b = torch.max(a, 0)[0]
c = torch.max(a, 1)[0]
print('a:',a)
print('b:',b)
print('c:',c)
```
输出为
```
a: tensor([[3, 5],
        [2, 3],
        [6, 7]])
b: tensor([6, 7])
c: tensor([5, 3, 7])
```
